clang/test/dpct/cmake_migration/case_038/input.cmake

```diff
@@ -0,0 +1,3 @@
+target_link_directories(CUDA::toolkit INTERFACE "${CUDAToolkit_LIBRARY_DIR}")
```

Comment:
do we need process CUDA::toolkit?

Reply:
No, it is just a target name, could be anything